dali/operators/math/expressions/expression_impl_gpu.cuh

```diff
@@ -101,6 +100,31 @@ __device__ void ExecuteBinOp(Result *result, const Left *l, Right r, int64_t ext
   }
 }
 
+/**
+ * @brief Loop over tile of `extent` length and apply the ternary op
+ */
+template <ArithmeticOp op, typename Result, bool IsFirstTensor, bool IsSecondTensor,
+          bool IsThirdTensor>
+__device__ void ExecuteTernaryOp(Result *result,
+                                 expression_detail::param2_t<IsFirstTensor, Result> first,
+                                 expression_detail::param2_t<IsSecondTensor, Result> second,
+                                 expression_detail::param2_t<IsThirdTensor, Result> third,
+                                 DALIDataType first_type, DALIDataType second_type,
+                                 DALIDataType third_type, int64_t extent) {
+  using meta = arithm_meta<op, GPUBackend>;
```

Comment:
```suggestion
  using Meta = arithm_meta<op, GPUBackend>;
```
or
```suggestion
  using meta_t = arithm_meta<op, GPUBackend>;
```
would read a little better. Just a suggestion

Reply:
Done, changed to `meta_`.